emukit/bayesian_optimization/loops/bayesian_optimization_loop.py

```diff
@@ -48,3 +56,33 @@ def __init__(self, model: IModel, space: ParameterSpace, acquisition: Acquisitio
         loop_state = create_loop_state(model.X, model.Y)
 
         super().__init__(candidate_point_calculator, model_updaters, loop_state)
+
+    def get_results(self):
+        return BayesianOptimizationResults(self.loop_state)
+
+
+class BayesianOptimizationResults:
```

Comment:
This class assumes one output dimension. I think this is OK for now but we will need a different class if we do multi-output BO in future.

Reply:
Sure, for the muti-output dimension case we'll have another loop (based on the notebook you wrote for multi-fidelity) so we can implement an specific method there